package.json

```diff
@@ -38,6 +44,11 @@
     ]
   },
   "devDependencies": {
-    "husky": "^7.0.0"
+    "husky": "^7.0.4"
+  },
+  "husky": {
```

Comment:
It looks like this way of doing the pre-commit hooks with husky has been deprecated as of version 7. I couldn't get it to run the prettier command. From the [docs](https://typicode.github.io/husky/#/?id=automatic-recommended), I found that you need to add to that `.husky/pre-commit` bash script instead. So I tested adding `npx --no-install lint-staged` to the end of the file and it worked like a charm. So you can probably remove this "husky" section from the `package.json` file